src/panel_gwalker/_pygwalker.py

```diff
@@ -0,0 +1,61 @@
+from typing import TYPE_CHECKING, Any, Dict, List
```

Comment:
Currently the `server` `computation` depends on PygWalker import because that is much easier than reinventing the wheel. We should figure out if that is ok? Or if we should carve out what we need.

The PygWalker license i Apache 2. We also need to consider if there are any issues carving out. I don't think so?